dev/breeze/src/airflow_breeze/utils/docker_command_utils.py

```diff
@@ -299,6 +299,38 @@ def check_docker_compose_version(verbose: bool):
         )
 
 
+def check_docker_context(verbose: bool):
+    """
+    Checks whether Docker is using the expected context
+    :param verbose: print commands when running
+    """
+    expected_docker_context = "default"
+    response = run_command(
+        ["docker", "info", "--format", "{{json .ClientInfo.Context}}"],
+        verbose=verbose,
+        no_output_dump_on_exception=False,
+        text=True,
+        capture_output=True,
+    )
+    if response.returncode != 0:
+        get_console().print(
+            '[error]Could not check for Docker context.[/]\n'
+            '[warning]Please make sure Docker is installed and running properly.[/]'
+        )
+
```

Comment:
as discused above - I think we should return here. Otherwise we are passing srdout of command that failed to parse the context - which might have strange results. What happen if stdout is None (I guess this wil happen if the process did not even manage to start) - but not sure about it. Regardless return here is better.

Reply:
Added a return :) 
It's making a lot of sense.